app/controllers/application_controller/ci_processing.rb

```diff
@@ -1619,6 +1619,42 @@ def scanhosts
     host_button_operation('scan', 'Analysis')
   end
 
+  # Scan all selected or single displayed image(s)
+  def scanimages
```

Comment:
@oschreib looks like this button is a copy of vm_button_operation method https://github.com/ManageIQ/manageiq/blob/master/app/controllers/application_controller/ci_processing.rb#L1102, is there a way to avoid duplicate code, refactor some of the code to a common method.